### ISSUE TITLE
Rescaled planets and moons displaying a ghosted surface

### ISSUE THREAD

**Issue by TykoTek** [Sigma88/Sigma-Dimensions#77]
I'm using Rescale 2.5x and GPP. When in orbit above planets and moons I'm seeing a "ghosted" image of a second terrain surface. On some bodies the height between the visible surface and the ghost images are more pronounced than others, but it's happening to all of them. The effect almost looks like a cloud layer but the clouds are in the shape of mountains. Of note, it looks like the ghost texture isn't aligned with the underlying hard surface - you may see a ghost mountain above a flat plain. 

Hard to capture on screen shot, but here are a few examples:
Note the lighter band of grey on the right edge where the light is shining through the ghost terrain:
![screenshot3](https://user-images.githubusercontent.com/22890228/46487773-b7b68400-c7b5-11e8-8fac-981b43b68aa4.png)

Here's another - note the lighter color that looks like an atmosphere, but it's terrain shapes
![screenshot8](https://user-images.githubusercontent.com/22890228/46487786-c1d88280-c7b5-11e8-95bc-57c040e02dac.png)

I'm using KSP 1.4.5 with GPP 1.6.3.1, Kopernicus 1.4.5-4, Rescale 2.5 1.0.2.8 and Sigma Dimensions 0.9.8 (I've also tried 0.10.1)

Thanks

**Comment by TykoTek** [comment]
That didn't fix it. I've tried creating an entirely new game install with just GPP, supporting mods and Rescale/SD and it still happens.

**Comment by Sigma88** [comment]
can you try reproduce the issue on a stock system + SD?

if it happens only with GPP that might be an interesting point

**Comment by TykoTek** [comment]
It's not occurring in Stock. As soon as I install GPP I'm seeing it. This was a brand new bare bones install. Only basic dependencies.

**Comment by Sigma88** [comment]
hmmm I don't know, I will check it out

could you send me the settings you use to get 2.5x?

**Comment by TykoTek** [comment]
I'm using Galileo's ReScale 2.5x and 3.2x no changes to the his default settings. 

https://github.com/Galileo88/Rescale

**Comment by Sigma88** [comment]
and you are not using scatterer, right?

**Comment by TykoTek** [comment]
Correct. I was not using scatterer in the clean test install. it was a brand new install with only these folders in GameData: GPP, Kopernicus, ModularFlightIntegrator, Rescale, SD, Squad, Squad expansion + Module Manager

**Comment by Sigma88** [comment]
I have enough for now to try and reproduce/fix the issue. I haven't had much time lately but I should get some in the near future.

you could try using other planet packs to see if the same issue occurs, for example you could try OPM which is made by galileo (same as OPM)

the interesting part of GPP is that it keeps the stock planets so you can see if all planets are affected or only those added by the pack

**Comment by Sigma88** [comment]
wait, is this your issue?

https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1028-03-dec-2017/&do=findComment&comment=3469445


if the problem is visible only at certain altitudes it's probably the change from scaledspace to pqs

I don't know if there are solutions to that, I'll have to look if I can do something, but it's not a SD issue

**Comment by TykoTek** [comment]
Yea, that's my issue. It only occurs in a middle altitude band. below 25km and no problem

Here's my GameData folder. it's a clean install
![image](https://user-images.githubusercontent.com/22890228/47091726-44136e80-d1da-11e8-86f8-f82f983eb609.png)

**Comment by Sigma88** [comment]
hmm you have EVE? try removing that

maybe it's something to do with that

**Comment by TykoTek** [comment]
LOL...you know, great minds think alike. Right after I posted I realized I could remove that too. I didn't have time to play around with it before work. I'll try it tonight.

**Comment by TykoTek** [comment]
Nope, removing EVE didn't change it

**Comment by Sigma88** [comment]
can you try with the new version?

**Comment by TykoTek** [comment]
Happy to help!


What environment should I be using? My previous tests were using ReScale, but I didn't think that would work with new Kopernicus.


Should I test this with ReScale & Kopernicus 1.7.0 or do I need to wait until ReScale is updated and test on 1.7.1?


TJ


TJ Truttmann


On June 8, 2019 at 3:34 AM, Sigma88 <notifications@github.com> wrote:



can you try with the new version?

**Comment by Sigma88** [comment]
rescale should work fine, with KSP 1.7.1, Kopernicus 1.7.1-4 and SigmaDimensions v0.10.3

**Comment by TykoTek** [comment]
just tested. It appears to occur over a narrower altitude range than before. That's great. In the attached video you can still see it when the craft is around 80km above the surface

https://youtu.be/_ym1meoEqGc

**Comment by Sigma88** [comment]
I think this effect is caused by the fading in-out of the PQS-ScaledSpace

maybe @Galileo88 can take a look to see if the cfgs of that body can be changed to reduce this effect

but it might just be a limitation of KSP itself
